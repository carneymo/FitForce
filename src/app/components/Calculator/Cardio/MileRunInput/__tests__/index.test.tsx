import * as React from 'react';
import { render } from '@testing-library/react';

import { MileRunInput } from '..';

describe('<MileRunInput  />', () => {
  it('should match snapshot', () => {
    const loadingIndicator = render(<MileRunInput age={''} gender={''} />);
    expect(loadingIndicator.container.firstChild).toMatchSnapshot();
  });
});
