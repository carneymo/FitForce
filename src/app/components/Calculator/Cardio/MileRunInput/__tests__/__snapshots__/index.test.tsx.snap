// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<MileRunInput  /> should match snapshot 1`] = `
<div
  class="MuiBox-root css-fgvrm7"
>
  <strong>
    1.5 Mile Run
  </strong>
  <div
    class="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2 css-mhc70k-MuiGrid-root"
  >
    <div
      class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 css-19gy2ei-MuiGrid-root"
    >
      <div
        class="MuiFormControl-root MuiFormControl-marginDense MuiTextField-root css-13e98gt-MuiFormControl-root-MuiTextField-root"
      >
        <label
          class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-sizeSmall MuiInputLabel-outlined MuiFormLabel-colorPrimary MuiFormLabel-filled MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-sizeSmall MuiInputLabel-outlined css-1jy569b-MuiFormLabel-root-MuiInputLabel-root"
          data-shrink="true"
          for="milerun-minutes"
          id="milerun-minutes-label"
        >
          mm
        </label>
        <div
          class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary MuiInputBase-formControl MuiInputBase-sizeSmall css-9ddj71-MuiInputBase-root-MuiOutlinedInput-root"
        >
          <input
            aria-invalid="false"
            class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputSizeSmall css-1n4twyu-MuiInputBase-input-MuiOutlinedInput-input"
            id="milerun-minutes"
            type="text"
            value="00"
          />
          <fieldset
            aria-hidden="true"
            class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
          >
            <legend
              class="css-14lo706"
            >
              <span>
                mm
              </span>
            </legend>
          </fieldset>
        </div>
      </div>
      <div
        class="MuiFormControl-root MuiFormControl-marginDense MuiTextField-root css-xnm3t8-MuiFormControl-root-MuiTextField-root"
      >
        <label
          class="MuiFormLabel-root MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-sizeSmall MuiInputLabel-outlined MuiFormLabel-colorPrimary MuiFormLabel-filled MuiInputLabel-root MuiInputLabel-formControl MuiInputLabel-animated MuiInputLabel-shrink MuiInputLabel-sizeSmall MuiInputLabel-outlined css-1jy569b-MuiFormLabel-root-MuiInputLabel-root"
          data-shrink="true"
          for="milerun-seconds"
          id="milerun-seconds-label"
        >
          ss
        </label>
        <div
          class="MuiInputBase-root MuiOutlinedInput-root MuiInputBase-colorPrimary MuiInputBase-formControl MuiInputBase-sizeSmall css-9ddj71-MuiInputBase-root-MuiOutlinedInput-root"
        >
          <input
            aria-invalid="false"
            class="MuiInputBase-input MuiOutlinedInput-input MuiInputBase-inputSizeSmall css-1n4twyu-MuiInputBase-input-MuiOutlinedInput-input"
            id="milerun-seconds"
            type="text"
            value="00"
          />
          <fieldset
            aria-hidden="true"
            class="MuiOutlinedInput-notchedOutline css-1d3z3hw-MuiOutlinedInput-notchedOutline"
          >
            <legend
              class="css-14lo706"
            >
              <span>
                ss
              </span>
            </legend>
          </fieldset>
        </div>
      </div>
    </div>
    <div
      class="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-6 css-fjbua9-MuiGrid-root"
    >
      <p
        class="MuiTypography-root MuiTypography-body1 css-e93pxh-MuiTypography-root"
      >
        5.0
      </p>
       
      <p
        class="MuiTypography-root MuiTypography-body1 css-vappet-MuiTypography-root"
      >
        / 20
      </p>
      <div
        class="MuiChip-root MuiChip-filled MuiChip-sizeSmall MuiChip-colorError MuiChip-filledError css-lwe8th-MuiChip-root"
      >
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiChip-icon MuiChip-iconSmall MuiChip-iconColorError css-i4bv87-MuiSvgIcon-root"
          data-testid="WarningAmberIcon"
          focusable="false"
          style="color: white;"
          viewBox="0 0 24 24"
        >
          <path
            d="M12 5.99 19.53 19H4.47L12 5.99M12 2 1 21h22L12 2z"
          />
          <path
            d="M13 16h-2v2h2zm0-6h-2v5h2z"
          />
        </svg>
        <span
          class="MuiChip-label MuiChip-labelSmall css-wjsjww-MuiChip-label"
        >
          Unsatisfactory
        </span>
      </div>
      <div
        class="MuiChip-root MuiChip-filled MuiChip-sizeSmall MuiChip-colorPrimary MuiChip-filledPrimary css-174hymm-MuiChip-root"
      >
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiChip-icon MuiChip-iconSmall MuiChip-iconColorPrimary css-i4bv87-MuiSvgIcon-root"
          data-testid="RecommendIcon"
          focusable="false"
          style="color: white;"
          viewBox="0 0 24 24"
        >
          <path
            d="M12 2a10 10 0 1 0 10 10A10 10 0 0 0 12 2zm6 9.8a.9.9 0 0 1-.1.5l-2.1 4.9a1.34 1.34 0 0 1-1.3.8H9a2 2 0 0 1-2-2v-5a1.28 1.28 0 0 1 .4-1L12 5l.69.69a1.08 1.08 0 0 1 .3.7v.2L12.41 10H17a1 1 0 0 1 1 1z"
          />
        </svg>
        <span
          class="MuiChip-label MuiChip-labelSmall css-wjsjww-MuiChip-label"
        >
          Satisfactory
        </span>
      </div>
      <div
        class="MuiChip-root MuiChip-filled MuiChip-sizeSmall MuiChip-colorSuccess MuiChip-filledSuccess css-li422f-MuiChip-root"
      >
        <svg
          aria-hidden="true"
          class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiChip-icon MuiChip-iconSmall MuiChip-iconColorSuccess css-i4bv87-MuiSvgIcon-root"
          data-testid="StarIcon"
          focusable="false"
          style="color: gold;"
          viewBox="0 0 24 24"
        >
          <path
            d="M12 17.27 18.18 21l-1.64-7.03L22 9.24l-7.19-.61L12 2 9.19 8.63 2 9.24l5.46 4.73L5.82 21z"
          />
        </svg>
        <span
          class="MuiChip-label MuiChip-labelSmall css-wjsjww-MuiChip-label"
        >
          Excellent
        </span>
      </div>
    </div>
  </div>
</div>
`;
